.travis.yml

```diff
@@ -0,0 +1,39 @@
+# Not using Python code, it's just a bone for Travis CI
+language: python
+python: 3.4
+
+sudo: required
+
+before_install:
+  - |
+    cat /etc/lsb-release | grep DISTRIB_DESCRIPTION && uname -a && whoami && pwd
```

Comment:
Most of the system information is automatically printed by Travis in `Build system information before this. See this [build](https://travis-ci.org/EA31337/FX-MT-VM/builds/84695306).